src/client/components/layer-list/layer-list.vue

```diff
@@ -69,11 +81,25 @@ export default {
   },
   data: () => ({
     expanded: '',
+    showLayerDialog: false,
+    maxHeight: 500,
   }),
+  mounted() {
+    this.calculateMaxHeight();
+  },
   methods: {
     setExpanded(id) {
       this.expanded = this.expanded !== id ? id : ''
     },
+    calculateMaxHeight() {
+      /**
+       * calculating possible max-height
+       * magic numbers: 300 as distance to top, 80 as spacing at bottom
+       */
+      // TODO/Extra: Add resize handler to recalculate maxHeight
+      const top = this.$refs.list ? this.$refs.list.$el.getBoundingClientRect().top : 300;
```

Comment:
Please write a ternary if statement like so:
```
condition
 ? true value
 : false value
```

It makes it more readable then having it on one line